astropy/io/votable/tests/tree_test.py

```diff
@@ -1,23 +1,26 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+import io
+import pytest
 
-from astropy.io.votable import exceptions
+from astropy.io.votable.exceptions import W07, W08, W21, W41
 from astropy.io.votable import tree
 from astropy.io.votable.table import parse
+from astropy.io.votable.tree import VOTableFile, Resource
 from astropy.tests.helper import raises
```

Comment:
Nice catch.  I didn't notice this at all.  It looks like there are more uses (via `@raises`) in `ucd_test.py` and `util_test.py` as well.  Should we clean those up with this PR or use a separate one?

Reply:
If you don't mind, we can attach them in this PR. Thanks! 😸

Reply:
@pllim Oops I missed this when the comments got collapsed.  I'll remove the remaining `helper.raises` later tonight when I'm back on-line unless you comment otherwise.

Reply:
Turns out there is another file outside of `votable`. I opened a PR already but please review #11670. 😉